lib/couchdb_to_sql/document_handler.rb

```diff
@@ -29,9 +29,9 @@ def handles?(doc)
     # Handle a table definition.
     def table(name, opts = {}, &block)
       if @mode == :delete
-        TableDestroyer.new(self, name, opts).execute
+        TableDestroyer.new(self, name, opts, &block).execute
```

Comment:
Not related to this change, but: `TableDestroyer` is really, _really_ a bad name of the class here. The casual reader will understand this as "destroy the given table", which is not at all what happens...

(I was probably the one who came up with that silly name to begin with.)